homeassistant/components/emulated_hue/__init__.py

```diff
@@ -280,35 +288,32 @@ def is_entity_exposed(self, entity):
             # Ignore entities that are views
             return False
 
-        domain = entity.domain.lower()
-        explicit_expose = entity.attributes.get(ATTR_EMULATED_HUE, None)
+        if entity.entity_id in self._entities_with_hidden_attr_in_config:
+            return not self._entities_with_hidden_attr_in_config[entity.entity_id]
+
         explicit_hidden = entity.attributes.get(ATTR_EMULATED_HUE_HIDDEN, None)
```

Comment:
Why not change this to a similar approach like hidden_attr.
```suggestion
        explicit_hidden = entity.attributes.get(ATTR_EMULATED_HUE_HIDDEN, None)
        if explicit_hidden is not None:
            return not explicit_hidden
```

Reply:
I can now that I completed the deprecated removal